### PR TITLE
Add 🤗 Transformers usage and info

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,47 @@ entropy coding, by clicking the thumbnail (original tracks provided by [Lucille
...
+
+# cast the audio data to the correct sampling rate for the model
+librispeech_dummy = librispeech_dummy.cast_column("audio", Audio(sampling_rate=processor.sampling_rate))
+audio_sample = librispeech_dummy[0]["audio"]["array"]
+
+# pre-process the inputs
+inputs = processor(raw_audio=audio_sample, sampling_rate=processor.sampling_rate, return_tensors="pt")
+
+# explicitly encode then decode the audio inputs
+encoder_outputs = model.encode(inputs["input_values"], inputs["padding_mask"])
```
Comment (adefossez) [inline README.md:71]: maybe add some comments on the shape of this, in particular how to recover the discrete codebook values, which will be of interest for anyone doing language modeling.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,47 @@ entropy coding, by clicking the thumbnail (original tracks provided by [Lucille
 <img src="./thumbnail.png" alt="Thumbnail for the sample video.
 	You will first here the ground truth, then ~3kbps, then 12kbps, for two songs."></a></p>
 
+## 🤗 Transformers
+
+Encodec has been added to Transformers. For more information, please refer to [Transformers' Encodec docs](https://huggingface.co/docs/transformers/main/en/model_doc/encodec).
+
+You can find both the [24KHz](https://huggingface.co/facebook/encodec_24khz) and [48KHz](https://huggingface.co/facebook/encodec_48khz) checkpoints on the 🤗 Hub.
+
+### Usage
```
Comment (adefossez) [inline README.md]: I'm worried about people getting confused with two `Usage` section. Maybe merge everything into one `Transformers` section.

File: README.md [hunk 3]
```diff
@@ -33,6 +33,47 @@ entropy coding, by clicking the thumbnail (original tracks provided by [Lucille
...
+
+You can use Encodec at scale with massive datasets with less than 10 lines of code! ⚡️
+
+Set up your development environment!
+```
+pip install datasets pip install git+https://github.com/huggingface/transformers.git@main
```
Comment (adefossez) [inline README.md]: missing line return

**Comment by Vaibhavs10** [comment]
Signed the CLA! I'm not quite sure why exactly is it still showing `red`, we might have to re-run the CLA check!

File: README.md [hunk 4]
```diff
@@ -33,6 +33,47 @@ entropy coding, by clicking the thumbnail (original tracks provided by [Lucille
...
 
+## 🤗 Transformers
+
+Encodec has been added to Transformers. For more information, please refer to [Transformers' Encodec docs](https://huggingface.co/docs/transformers/main/en/model_doc/encodec).
+
+You can find both the [24KHz](https://huggingface.co/facebook/encodec_24khz) and [48KHz](https://huggingface.co/facebook/encodec_48khz) checkpoints on the 🤗 Hub.
+
+### Usage
+
+You can use Encodec at scale with massive datasets with less than 10 lines of code! ⚡️
```
Comment (adefossez) [inline README.md]: Maybe rephrase as "Using Transformers smiley, you can use Encodec at scale along with all the other supported model and datasets. Alternatively you can also directly the `encodec` package, as detailed in the Usage section."

While I like Transformers I don't want to make it look like it is the only option! For some use cases the original encodec might be much lighter to install and use!

**Review by adefossez — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Vaibhavs10) [reply README.md]: Makes sense! Fixed!

File: README.md (hunk 3 above)
Comment (Vaibhavs10) [reply README.md]: oooh! good catch! fixed ty!

File: README.md (hunk 4 above)
Comment (Vaibhavs10) [reply README.md]: Definitely! I just copied your phrasing 1:1 with minor changes, hope that works!

File: README.md (hunk 1 above)
Comment (Vaibhavs10) [reply README.md:71]: Fantastic suggestion! I'll find the best way to showcase this and update it in a sec!

Comment (Vaibhavs10) [reply README.md:71]: Added the info for it!

**Review by adefossez — APPROVED** [review]
